docs/workflows.md

```diff
@@ -10,7 +10,7 @@ It's possible to download data from a remote server using a browser APIs or a fe
 
 #### CORS problems
 
-One of the most common problems people encounter when attempting to load data into Iodide from a remote server are issues with [CORS settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS), which can restrict the ability of Iodide to download data from some other websites. Unfortunately, for security reasons browsers make it [impossible to surface CORS errors in a web app](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Functional_overview), so Iodide cannot give you the guidance you need directly, but if you have trouble loading a data file from a third party server, this is very likely to be the reason why. To check, you can open your browser's developer tools (press `ctrl+shift+i`); in case of a CORS issue, you will see a message like: 
+One of the most common problems people encounter when attempting to load data into Iodide from a remote server are issues with [CORS settings](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS), which can restrict the ability of Iodide to download data from some other websites. Unfortunately, for security reasons browsers make it [impossible to surface CORS errors in a web app](https://developer.mozilla.org/en-US/docs/Web/HTTP/CORS#Functional_overview), so Iodide cannot give you the guidance you need directly, but if you have trouble loading a data file from a third party server, this is very likely to be the reason why. To check, you can open your browser's developer tools (press `ctrl+shift+i`); in case of a CORS issue, you will see a message like:
```

Comment:
???

Reply:
Oh, it removed a trailing space after *like:*. Weird that GitHub didn't highlight that.